integration-tests/manual-tests.md

```diff
@@ -241,3 +241,22 @@ $ snap remove bluez
           Vendor Specific data: 0000
           TPM Version:         01010000
           Manufacturer Info:   57454300
+
+# Test fwupd interface with uefi-fw-tools
+
+1. Ensure your BIOS support UEFI firmware upgrading via UEFI capsule format
+2. Install the uefi-fw-tools snap from the store
+3. Ensure the 'fwupd' interface is connected
```

Comment:
IIRC, fwupd does fetch things from over the internet, so it will always need 'network'. However, it would be great if:
- you could remove all other plugs for fwupdmgr and list the needed syscalls here (and ideally a general idea why they are needed-- but don't waste time on that-- just a high level (eg, it "binds to a port", "it calls out over the network on launch", etc. These calls may be added to PermanentSlotSecComp
- Now that the daemon has started, have the cli command talk to fwupdmgr and list those calls required for fwupdmgr to talk to the client. These would go in ConnectedSlotSecComp
- you could remove all other plugs from the cli command and talk to fwupdmgr and list those calls. These would go in ConnectedPlugSecComp.